include/MidiEvent.h

```diff
@@ -204,7 +218,9 @@ class MidiEvent
 
 	const char* m_sysExData;
 	const void* m_sourcePort;
-
+	bool m_ignoreOnExport;	// Should we ignore this MIDI event when exporting a project?
+				// This helps us ignoring MIDI events that shouldn't be processed
+				// during a project export, like physical controller events.
```

Comment:
Two minor problems: alignment with tabs and grammar. I think flipping the comment to above would be better, and some IDEs like vscode (maybe the C/C++ extension though) will use comments above as description of variable if no doctext is found.
```suggestion
	// This helps us ignore MIDI events that shouldn't be processed
	// during a project export, like physical controller events.
	bool m_ignoreOnExport; // Should we ignore this MIDI event when exporting a project?
```